src/components/messenger/list/group-details-panel/index.tsx

```diff
@@ -54,10 +54,13 @@ export class GroupDetailsPanel extends React.Component<Properties, State> {
           <ImageUpload onChange={this.onImageChange} icon={this.renderImageUploadIcon()} />
 
           <Input
-            value={this.state.name}
+            value={name}
             onChange={this.nameChanged}
-            {...cn('name-input')}
             placeholder='Group name...'
+            isRequired={true}
+            error={!this.isNameValid}
+            alert={!this.isNameValid ? { variant: 'error', text: 'Please enter a group name.' } : undefined}
```

Comment:
I think Dom is right here, I misunderstood the original problem. We can just keep the button disabled and as soon as the user types a name, we can enable it.  this code can be removed